src/midi.cpp

```diff
@@ -84,10 +100,37 @@ static void HandleNoteOn(byte channel, byte note, byte velocity) {
 
 static void HandleNoteOff(byte channel, byte note) { HandleNoteOn(channel, note, 0); }
 
+byte lastData1, lastData2; // keep track of last CC and Value for handshake with tool.
```

Comment:
```suggestion
static byte lastData1, lastData2; // keep track of last CC and Value for handshake with tool.
```

Make it static. This kinda acts like `private` and won't let someone from outside declare `extern byte lastData1` and use it. We want to keep our variables by ourselves ;)